docs/README.md

```diff
@@ -25,17 +25,15 @@ This is intended to coincide with the [Drupal core release cycle](https://www.dr
 
 ### Release support status
 
-| BLT version | Support status        | End of life    |  Drupal versions* | Drush versions         |
-|-------------|-----------------------|----------------|-------------------|------------------------|
-| 12.x        | Unsupported, unstable | \>=May 2021    | 9.0               | \>=10.0.1<br>\>=9.5.0    |
-| **11.x**    | **Supported, stable** | **>=Dec 2020** | **8.7\*\*, 8.8**  | **>=10.0.1**<br>\>=9.5.0 |
-| 10.x        | Bug fixes only        | May 2020       | 8.7, 8.8          | \>=9.5.0               |
-| <=9.2.x     | Unsupported           | Dec 2019       | 8.6, 8.7          | \>=9.4.0               |
+| BLT version | Support status        | End of life    |  Drupal versions*   | Drush versions           |
+|-------------|-----------------------|----------------|---------------------|--------------------------|
+| **12.x**    | **Supported, stable** | **\>=May 2021**| **9.0**             | **\>=10.0.1**<br>\>=9.5.0|
+| **11.x**    | **Supported, stable** | **>=Dec 2020** | **8.8, 8.9 LTS**    | **>=10.0.1**<br>\>=9.5.0 |
+| 10.x        | Bug fixes only        | May 2020       | 8.8                 | \>=9.5.0                 |
```

Comment:
For `10.x`, the EOL was May of this year. Does that mean that it still gets bug fixes, as stated in the second column?